botocove/cove_session.py

```diff
@@ -44,21 +43,6 @@ def activate_cove_session(self) -> "CoveSession":
             f"{self.session_information['RoleName']}"
         )
 
-        if self.org_master:
-            try:
-                account_description: AccountTypeDef = self.org_client.describe_account(
```

Comment:
I think the org_client could be entirely removed now from the CoveSession class. I don't see it used anywhere. Is there another reason to keep it?

Reply:
I doubt it, I intentionally avoided any deprecation or non performance clean up in this PR though